backend/src/main/java/de/lindatroesken/backend/service/MangelService.java

```diff
@@ -43,4 +43,30 @@ public MangelEntity createMangel(String username, MangelEntity newMangelEntity)
     public MangelEntity findMangelById(Long id) {
         return mangelRepository.findById(id).orElseThrow(() -> new IllegalArgumentException("Mangel not found"));
     }
+
+    public MangelEntity updateMangel(Long id, MangelEntity changedMangel) {
+        MangelEntity existingMangel = mangelRepository.findById(id).orElseThrow(() -> new EntityNotFoundException("Mangel not found"));
```

Comment:
ich habe die gefundene entity immer in  eine neue kopiert, diese dann bearbeitet und die kopie gesaved. ich glaube es kann probleme verursachen wenn du auf der orginal entity ein .set()  machst da JPA eventuell versucht das sofort in der datenbank zu ändern

Reply:
ok. dann kommt hier noch ein Zwischenschritt hin. Bisher hat es funktioniert, aber sicher ist sicher.

Reply:
Mit dem Zwischenschritt kopieren bekomme ich gerade Fehlermeldungen. Lasse ich erstmal so und schaue ich mir später an.